Prox/Prox/Widgets/Prompts.swift

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+/// A collection of random prompt functions.
+/// Maybe these would be better as a UIAlertController subclass. If so, make sure you continue to
+/// log the analytics in a way people won't forget to call it (e.g. in `viewWillAppear`).
+struct Prompts {
```

Comment:
> Is your concern that putting it in the file directly makes that file too big?

Precisely. I've been finding it's much easier to quickly & confidently edit a file if all of the things that don't concern what I'm trying to do are encapsulated away into their own files (e.g. these Prompts have nothing to do with an particular VC) & the file isn't too large.

> A generic prompt container doesn't really make sense as under Widget/ to me, where I consider "widget" to be a general-purpose, reusable UIView subclass. Are you using widgets for different things in Prox?

That's generally what I consider a widget but I didn't know where else to put these. Perhaps a top-level Prompts file or `/Prompts/` with a separate file for each?